workspaces/orchestrator/plugins/orchestrator/README.md

```diff
@@ -86,6 +86,21 @@ orchestrator:
 
 For more information about the configuration options, including other optional properties, see the [config.d.ts](../orchestrator-common/config.d.ts) file.
 
+#### How to use the Git Hub identity provider
```

Comment:
This should move the the parent README, these relative links are incorrect in this context

Reply:
Since this README describes the whole setup, partially including documentation for other `orchestrator-*` plugins, I would keep the info co-located and fix the paths only.

Or let's move even the rest to the `/workspaces/orchestrator/README.md` as well.

Reply:
ok let's move it to /workspaces/orchestrator/README.md since it contains info also on the other plugins

Reply:
so...
Are you moving the whole README to workspaces/orchestrator/README?

Reply:
yes